.gitattributes

```diff
@@ -4,6 +4,8 @@
 * text eol=lf
 *.cs eol=lf
 *.csproj eol=lf
+*.props eol=lf
```

Comment:
Is there any .props file in the plugins being merged?

Reply:
yes have 3 different ones